transaction.go

```diff
@@ -304,7 +315,9 @@ func (t *Transaction) SignPayload(address Address, keyIndex int, signer crypto.S
 //
 // This function returns an error if the signature cannot be generated.
 func (t *Transaction) SignEnvelope(address Address, keyIndex int, signer crypto.Signer) error {
-	sig, err := signer.Sign(t.EnvelopeMessage())
+	message := t.EnvelopeMessage()
+	message = append(t.TransactionDomainTag, message...)
+	sig, err := signer.Sign(message)
```

Comment:
The comment doesn't belong to the code above:
Does the SDK also implement verifications for the payload and envelope signatures? If yes, then the tag needs to be prepended to the message in those functions as well.

Reply:
No, the Go SDK doesn't have any transaction signature verification functionality.

Reply:
Thanks @Peter!
Do you know if there is a flow-go issue to mirror this SDK change in the verifications on the FVM side  ?

Reply:
@tarakby this is the change in flow-go https://github.com/onflow/flow-go/pull/606